proposed/simplecache.md

```diff
@@ -114,85 +112,117 @@ namespace Psr\SimpleCache;
+     *
+     * @throws InvalidArgumentException
+     *   If the $key string is not a legal value a \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown.
      */
     public function set($key, $value, $ttl = null);
 
     /**
-     * Delete an item from the cache by its unique key
+     * Delete an item from the cache by its unique key.
+     *
+     * @param string $key The unique cache key of the item to delete.
      *
-     * @param string $key The unique cache key of the item to delete
+     * @return bool True if the item was successfully removed. False if there was an error.
      *
-     * @return void
+     * @throws InvalidArgumentException
+     *   If the $key string is not a legal value a \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown.
      */
     public function delete($key);
 
     /**
-     * Wipe clean the entire cache's keys
+     * Wipes clean the entire cache's keys.
      *
-     * @return void
+     * @return bool True on success and false on failure.
      */
     public function clear();
 
     /**
-     * Obtain multiple cache items by their unique keys
+     * Obtains multiple cache items by their unique keys.
+     *
+     * @param array|\Traversable $keys    A list of keys that can obtained in a single operation.
+     * @param mixed              $default Default value to return for keys that do not exist.
      *
-     * @param array|Traversable $keys A list of keys that can obtained in a single operation.
+     * @return array|\Traversable A list of key => value pairs. Cache keys that do not exist or are stale will have $default as value.
      *
-     * @return array An array of key => value pairs. Cache keys that do not exist or are stale will have a value of null.
+     * @throws InvalidArgumentException
+     *   If $keys is neither an array nor a Traversable, or
+     *   if any of the keys in $keys are not a legal value a \Psr\SimpleCache\InvalidArgumentException
+     *   MUST be thrown.
      */
-    public function getMultiple($keys);
+    public function getMultiple($keys, $default = null);
 
     /**
-     * Persisting a set of key => value pairs in the cache, with an optional TTL.
+     * Persists a set of key => value pairs in the cache, with an optional TTL.
      *
-     * @param array|Traversable     $items An array of key => value pairs for a multiple-set operation.
+     * @param array|\Traversable    $items A list of key => value pairs for a multiple-set operation.
      * @param null|int|DateInterval $ttl   Optional. The TTL value of this item. If no value is sent and
      *                                     the driver supports TTL then the library may set a default value
      *                                     for it or let the driver take care of that.
      *
-     * @return bool True on success and false on failure
+     * @return bool True on success and false on failure.
+     *
+     * @throws InvalidArgumentException
```

Comment:
@throws FQCN ..

Then: `If $keys is neither an array nor a Traversable, or if any of the $keys are not a legal value.`